[17/5/2025, 19:19:58] CRITICAL: Fallo al iniciar el servidor o conectar a la base de datos {"codigo_error":"SERVER_START_FAIL","fecha_hora":"17/5/2025, 19:19:58","stack":"SequelizeConnectionError: Connection terminated unexpectedly\n    at Client._connectionCallback (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\dialects\\postgres\\connection-manager.js:149:20)\n    at Connection.<anonymous> (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\pg\\lib\\client.js:144:18)\n    at Object.onceWrapper (node:events:632:28)\n    at Connection.emit (node:events:518:28)\n    at Socket.<anonymous> (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\pg\\lib\\connection.js:62:12)\n    at Socket.emit (node:events:530:35)\n    at TCP.<anonymous> (node:net:343:12)","ip_servidor":"localhost"}
